README.md

```diff
@@ -1,26 +1,78 @@
+<!-- markdownlint-disable MD026 MD033 -->
+
 # DiscordFeedback
-The bot that powers the Discord Feedback server
 
-# Selfhosting
-We don't (fully) support hosting your own copy of DiscordFeedback, this program is highly customized and not suited to run outside of the official Discord Feedback server.
+This is the bot that powers the Discord Feedback server, develop'd by the volunteer group only known as "Discord Feedback Bot Tinkerers Sqaud™"
+
+## Table of Contents
+
+- [Development](#development)
+  - [Config](#config)
+  - [Global ESlint Install](#global-eslint-instal)
+  - [Testing](#coming-soon)
+- [Selfhosting](#selfhosting)
+  - [Config Creation](#config-creation)
+  - [Database Initalization](#database-initialization)
+  - [Running the Bot](#running-the-bot)
+
+## Development
+
+We here at Discord Feedback Bot Tinkerers Sqaud™ use Visual Studio Code, with the [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint), [npm](https://marketplace.visualstudio.com/items?itemName=eg2.vscode-npm-script), [npm Intellisense](https://marketplace.visualstudio.com/items?itemName=christian-kohler.npm-intellisense) plugins in order to produce the bestest code.
+
+### Config
+
+We like to keep things simple, when working on the bot just add ``debug: true,`` to the config, right after the first line
+
+```js
+
+module.exports = {
+  debug: true,
+  discord: {...}
+}
+
+```
+
+### Global ESlint Install
+
+ESlint has the ability to be install globally (allows for cmd line useage, which is helpful for coding), however since we use a few plugins for ESlint that usage won't work out of the box, so you'll need to install them. We've had luck using
+
+```batch
 
-However, if you do want to run an instance, you need the following:
+npm i -g eslint eslint-plugin-standard@latest eslint-plugin-promise@latest eslint-plugin-import eslint-config-standard@latest eslint-plugin-node@latest
```

Comment:
🤔

Reply:
yikes that's a mess, I'll fix that in a bit.